lib/compat/wordpress-6.2/script-loader.php

```diff
@@ -165,6 +165,8 @@ function( $settings ) {
 		// We must override what core is passing now.
 		$settings['__unstableResolvedAssets']    = gutenberg_resolve_assets_override();
 		$settings['__unstableIsBlockBasedTheme'] = wp_is_block_theme();
+		// As the name suggests, this shouldn't be backported to core.
+		$settings['__unstableIsGutenbergPlugin'] = IS_GUTENBERG_PLUGIN;
```

Comment:
Do you need it as a setting for some reason? 🤔 Why not use the ` IS_GUTENBERG_PLUGIN` from `process.env` in JS?